inc/dwarf.h

```diff
@@ -252,6 +252,7 @@ class Dwarf : public QObject
     Q_INVOKABLE int kinesthetic_sense() {return attribute(AT_KINESTHETIC_SENSE);}
     //! attribute value from id
     Q_INVOKABLE int attribute(ATTRIBUTES_TYPE attrib_id) {return get_attribute(attrib_id).get_value();}
+	Q_INVOKABLE int attribute_maximum(ATTRIBUTES_TYPE attrib_id) {return (int)get_attribute(attrib_id).max();}

```

Comment:
A bit nitpicky, but whitespace doesn't match here (you used a tab instead of 4 spaces).